classes/announcement/PKPAnnouncementDAO.inc.php

```diff
@@ -339,6 +339,7 @@ function &getAnnouncementsNotExpiredByAssocId($assocType, $assocId, $rangeInfo =
 			WHERE assoc_type = ?
 				AND assoc_id = ?
 				AND (date_expire IS NULL OR date_expire > CURRENT_DATE)
+				AND (date_posted < CURRENT_DATE)
```

Comment:
is not null would make more sense right? Can we set the date posted to anything higher than  the current date?